wasm-node/javascript/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
+        "target": "ES6",
```

Comment:
This is already in the two other `tsconfig` JSON files, there shouldn't be any need to repeat it here.

```suggestion
```

Reply:
Oops, an oversight on my part, thanks for pointing this out, have removed accordingly 🙏